app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,12 @@
   <%= stylesheet_link_tag 'application', media: 'all' %>
   <%= csrf_meta_tags %>
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/foundation-sites@6.7.5/dist/css/foundation.min.css" crossorigin="anonymous">
+  <link rel="stylesheet" href="app/assets/stylesheets/custom.css">
```

Comment:
importing stylesheets is already handled for us by the `stylesheets/application.css` file, so let's remove this line!